### ISSUE TITLE
Increase the GCB quota limit

### ISSUE THREAD

**Issue by cpanato** [kubernetes/k8s.io#1576]
#### What happened:

When some post jobs to build the images ran all together we are getting some quota limit in the GCB

```
ERROR: (gcloud.builds.submit) RESOURCE_EXHAUSTED: Quota exceeded for quota metric 'Build and Operation Get requests' and limit 'Build and Operation Get requests per minute' of service 'cloudbuild.googleapis.com' for consumer 'project_number:180382678033'.
```

Project number: `180382678033`

We need to increase the limit in the google cloud platform for the google cloud build to allow more jobs to be submitted in parallel.

Parent Issue: https://github.com/kubernetes/release/issues/1869

cc: @kubernetes/release-engineering @kubernetes/release-team-leads 
/assign @spiffxp

**Comment by spiffxp** [comment]
Looking...

https://cloud.google.com/cloud-build/quotas only lists
- concurrent builds per project (10)
- build triggers per project (300)

180382678033 is... k8s-infra-prow-build-trusted
```
$ gcloud projects describe 180382678033
createTime: '2020-04-30T23:40:44.839Z'
lifecycleState: ACTIVE
name: k8s-infra-prow-build-trusted
parent:
  id: '758905017065'
  type: organization
projectId: k8s-infra-prow-build-trusted
projectNumber: '180382678033'
```

This page should be visible to members of k8s-infra-prow-oncall@kubernetes.io (ref: https://github.com/kubernetes/k8s.io/blob/master/groups/groups.yaml#L628)
https://console.cloud.google.com/iam-admin/quotas?orgonly=true&project=k8s-infra-prow-build-trusted&supportedpurview=project

Navigating through it I see yeah, we're exceeding quota
![Screen Shot 2021-01-25 at 6 17 58 PM](https://user-images.githubusercontent.com/49258/105778119-a91da900-5f39-11eb-9e5c-1e75865d2101.png)

/assign @ameukam @cpanato 
Now I would like help understanding why we're exceeding quota, to understand what we should bump it to:
- was this a spike in traffic?
- is this due to a trend in traffic?
- why aren't these requests being charged against the projects running the builds?

**Comment by spiffxp** [comment]
None of the other cloud build quotas for this project are anywhere near their limit

**Comment by cpanato** [comment]
@spiffxp 
I saw that happening when some image build jobs have triggered all together (those jobs: https://github.com/kubernetes/test-infra/blob/master/config/jobs/image-pushing/releng/k8s-staging-build-image.yaml)

For example:
- https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/post-release-push-image-debian-hyperkube-base/1352583162915459072

- was this a spike in traffic?
yes, most likely, because all jobs were trigger at same time 

- is this due to a trend in traffic?
No. I don't think this is a trend.

- why aren't these requests being charged against the projects running the builds?
I think this is reaching the quota to get a build, like streaming the logs and maybe other things related to the cloud build job. I did not find any documentation over GCP docs to explain that and the quota page also does not explain. however, per the quota description, I think it is related to getting a build.
but it just my thought :)
/shrug

**Comment by spiffxp** [comment]
We're going to continue to hit this if we don't refactor which project requests are being allocated to.

I was going to bump quota while we investigate further, but it turns out I can only lower quota for this, not raise it.  This is a service api limit
![Screen Shot 2021-01-26 at 11 56 59 AM](https://user-images.githubusercontent.com/49258/105877102-9b196800-5fcd-11eb-9f39-a0f8c98691e2.png)

k8s-infra-prow-viewers@kubernetes.io can see this:
https://console.cloud.google.com/apis/api/cloudbuild.googleapis.com/quotas?orgonly=true&project=k8s-infra-prow-build-trusted&supportedpurview=project&pageState=(%22duration%22:(%22groupValue%22:%22P30D%22,%22customValue%22:null))
![Screen Shot 2021-01-26 at 12 07 32 PM](https://user-images.githubusercontent.com/49258/105878458-13345d80-5fcf-11eb-91e0-9755a25c1134.png)

30d smooths over peaks but shows general trend, the important thing is the quota exceeded errors going non-zero

**Comment by spiffxp** [comment]
If I had to guess off-the-cuff, I wonder if the gcb jobs should be using a service account per staging project, instead of one shared 'gcb-builder' service account.  Are there some central shared resources that require the gcb-builder SA?

**Comment by spiffxp** [comment]
Out of curiosity I checked out k8s-staging-e2e-test-images since we think that's the culprit for recent spikes:
https://console.cloud.google.com/apis/api/cloudbuild.googleapis.com/quotas?orgonly=true&project=k8s-staging-e2e-test-images&supportedpurview=project&pageState=(%22duration%22:(%22groupValue%22:%22P14D%22,%22customValue%22:null))

No api request traffic, concurrent build quota exceeded sometime on 2021-01-20.

Concurrent build quota is also something I can only lower, not raise.

**Comment by spiffxp** [comment]
/close
I'm going to close this as "can't do" and open a separate issue for https://github.com/kubernetes/k8s.io/issues/1576#issuecomment-767694092

**Comment by cpanato** [comment]
closing this as well since the parent in k/k8s.io is closed as well https://github.com/kubernetes/k8s.io/issues/1576#issuecomment-769421693

/close
